Dockerfile-linux.template

```diff
@@ -159,12 +159,20 @@ ENV GPG_KEYS {{
 		# https://www.php.net/gpg-keys.php
 		# https://www.php.net/downloads.php
 
+		"8.4": [
+			# https://wiki.php.net/todo/php84#release_managers
+			# https://www.php.net/gpg-keys.php#gpg-8.4
+			"AFD8 691F DAED F03B DF6E  4605 63F1 5A9B 7153 76CA", # ericmann
+			"9D7F 99A0 CB8F 05C8 A695  8D62 56A9 7AF7 600A 39A6", # calvinb
+			"0616 E93D 95AF 4712 43E2  6761 7704 26E1 7EBB B3DD"  # saki
+		],
+
 		"8.3": [
 			# https://wiki.php.net/todo/php83#release_managers
 			# https://www.php.net/gpg-keys.php#gpg-8.3
-			"1198 C011 7593 497A 5EC5  C199 286A F1F9 8974 69DC",  # pierrick
-			"C28D937575603EB4ABB725861C0779DC5C0A9DE4", # bukka
-			"AFD8 691F DAED F03B DF6E  4605 63F1 5A9B 7153 76CA" # ericmann
+			"1198 C011 7593 497A 5EC5  C199 286A F1F9 8974 69DC", # pierrick
+			"C28D937575603EB4ABB725861C0779DC5C0A9DE4",           # bukka
```

Comment:
Oh wild, I guess this was probably just to match the format on the linked page specifically because we can. :smile: :+1: